doc/reference/pipeline_spec.md

```diff
@@ -151,6 +153,26 @@ create pipeline](../pachctl/pachctl_create_pipeline.html) doc.
   "branch": string
 }
 
+------------------------------------
+"spout"
+------------------------------------
+
+"env": {
+  "HOST": int,
+  "PORT": int
+},
+
+"spout": {
+  "overwrite": bool
+  \\ Optionally, you can combine a spout with a service:
+  "service": {
+        "internal_port": int,
+        "external_port": int,
+        "annotations": {
+            "foo": "bar"
+        }
+    }
+}
```

Comment:
@kevindelgado , can you please confirm that this is correct or needs changes?

Reply:
I think it is confusing to place it here with the list `input` options, because it makes it seem like a spout is an type of `input` (which it is not, it is just an optional field on the top-level object. So I would place it immediately after the service object (line 78).

I'm also not sure what the `env` object here is, can you clarify where this is from?

Reply:
I think he approved above